src/components/BulkEnrollmentResultsDownloadPage/index.jsx

```diff
@@ -37,16 +36,38 @@ const BulkEnrollmentResultsDownloadPage = ({ enterpriseId }) => {
     }
   }, [bulkEnrollmentJobId, enterpriseId, isLoading]);
 
+  useEffect(() => {
+    setShowToast(true);
```

Comment:
Should there be some sort of conditional around `setShowToast` here? As is, it will get set to `true` any time either `notReady` or `error` changes, regardless if `notReady` is true or false.